packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -219,6 +220,18 @@ describe('c-quantic-search-box-input', () => {
               );
             expect(suggestionsListItems).not.toBeNull();
             expect(suggestionsListItems.length).toEqual(mockSuggestions.length);
+
+            const suggestionOptionLabels =
+              suggestionsList.shadowRoot.querySelectorAll(
+                selectors.suggestionOptionText
+              );
+            const suggestionsLength = mockSuggestions.length;
+
+            expect(suggestionOptionLabels).not.toBeNull();
+            expect(suggestionOptionLabels.length).toEqual(suggestionsLength);
+            expect(suggestionOptionLabels[suggestionsLength - 1].title).toEqual(
+              mockSuggestions[suggestionsLength - 1].value
+            );
```

Comment:
You can't just check the length or the title of the last one.


```suggestion
mockSuggestions.forEach((suggestion, idx) => {
  expect(suggestionOptionsLabels[idx].title).toEqual(suggestion.value);
});
```

Reply:
@SimonMilord this was not fixed?